src/airdrops/2-UNICORN.json

```diff
@@ -0,0 +1,42 @@
+{
+  "info": {
+    "name": "Unicorn",
+    "symbol": "UNICORN",
+    "contractAddress": "",
+    "network": "Cosmos",
+    "description": "ancient Windows 98 wizards, and neckbeard-fedora-wearing warriors",
+    "image": "https://unicorn.meme/img/uni-head.webp"
+  },
+  "claim": {
+    "dates": {
+      "claimDateOpen": "2024-04-02T00:00:00.000Z",
+      "claimDateClose": null,
+      "claimDuration": "unknown"
+    },
+    "value": {
+      "airdroppedTokens": null,
```

Comment:
i don't know any of this. `null` violates the typing in `src/lib/types.ts` but nothing appears to have broken? lmk how you want me to handle

Reply:
I made the types attributes more explict with the null field and can be safe to check against that in the template.